app/src/docs/content/streams/code/streams.js

```diff
@@ -1,11 +1,12 @@
 export const CreateJavascriptClient =
-`const client = new StreamrClient({
+`const streamr = new StreamrClient({
```

Comment:
Is private key auth the preferred method for authenticating? You can easily login with 

```js
const streamr = new StreamrClient({
    auth: {
        provider: web3.currentProvider,
    },
})
```

This actually handles or the mechanics described in API / Autherntication to get the token.

Reply:
And this bit of code is missing from the authentication section, maybe worth adding there at some ponit.

Reply:
Good point, I'll add it in. 
 
I *think* `web3.currentProvider` -> `window.ethereum` these days?

Reply:
Yeah, that should be the preferred one

> MetaMask injects a global API into websites visited by its users at window.ethereum (Also available at window.web3.currentProvider for legacy reasons).